docs/integrate/hivemq/usage.md

```
@@ -0,0 +1,72 @@
+```psql
++-------------+----------+
+| temperature | humidity |
++-------------+----------+
+|       42.84 |     83.1 |
++-------------+----------+
+SELECT 1 row in set (0.004 sec)
+```
+
+
+[HiveMQ MQTT Broker]: https://www.hivemq.com/products/mqtt-broker/
+[LorryStream]: https://lorrystream.readthedocs.io/
```

markdownlint-cli2 (0.18.1)

Link and image reference definitions should be needed
Unused link or image reference definition: "lorrystream"

(MD053, link-image-reference-definitions)